test/e2e/tests/custom-rpc-history.spec.js

```diff
@@ -63,7 +63,7 @@ describe('Custom RPC history', function () {
           '.networks-tab__add-network-form-footer .btn-primary',
         );
 
-        await driver.findElement({ text: networkName, tag: 'span' });
+        await driver.findElement({ text: networkName, tag: 'h6' });
```

Comment:
I'm guessing this `h6` is the "New network added" message (which includes the network name we're selecting here).

What was it before? :thinking:

Reply:
Oh I see, it was from when `setActive` was true, so the `new-network-info` modal was being shown. That modal has the network name in a span.